src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -78,6 +78,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\CommunityToolkit.Maui.SourceGenerators\CommunityToolkit.Maui.SourceGenerators.csproj" OutputItemType="Analyzer" ReferenceOutputAssembly="false" />
```

Comment:
can you add this line inside the `PropertyGroup`?
`<CompilerGeneratedFilesOutputPath>$(BaseIntermediateOutputPath)Generated</CompilerGeneratedFilesOutputPath>`

This will create a folder inside `obj` and we can inspect the generated file there... That's useful because sometimes VS stops to generate files and you need to restart it to see it under solution explorer

Reply:
Ah - nice! I read that the generated files should appear in `obj`, but I couldn't find them 😆

Reply:
Not sure if you know, but you can see it on VS as well... 
![image](https://user-images.githubusercontent.com/20712372/146629748-e8d8df88-f776-4c60-88af-54995ea62760.png)

Reply:
I still don't see the files listed in the `Analyzers` drop down. 

Did I implement this correctly? https://github.com/CommunityToolkit/Maui/pull/224/commits/adf99bf15468dc70064aac5b5af14bda62912a1c

Reply:
@brminnick try to re-open the VS

Reply:
<img width="399" alt="image" src="https://user-images.githubusercontent.com/13558917/146653345-593347fe-09be-4f79-a952-f81654e8bbac.png">
Coooool!